src/Nova/Location.php

```diff
@@ -48,7 +48,9 @@ public function fields(Request $request)
             Text::make('Abbreviation')
                 ->withMeta(['extraAttributes' => ['maxlength' => 4]])
                 ->required(),
-            nova('timezone') ? BelongsTo::make('Timezone', 'timezone', nova('timezone'))->required() : null,
+            nova('timezone') ? BelongsTo::make('Timezone', 'timezone', nova('timezone'))
```

Comment:
Not quite sure if there's a way to auto-update this once a Market is selected, so for now I just made the field optional and added a label that indicates the field will be populated if nothing else is selected.

<img width="759" alt="Screen Shot 2021-04-12 at 8 32 49 AM" src="https://user-images.githubusercontent.com/24723492/114402664-ac542880-9b69-11eb-8c38-f1be12af3c78.png">

Reply:
Yeah, that's perfect. Thanks!